src/components/editMeasure/details/MeasureMetaDataRow.tsx

```diff
@@ -10,10 +10,11 @@ interface MeasureMetaDataRowProps {
   description: string;
   id?: string;
   handleEdit?: Function;
+  canEdit?: boolean;
 }
 
 const MeasureMetaDataRow = (props: MeasureMetaDataRowProps) => {
-  const { name, description, id, handleEdit } = props;
+  const { name, description, id, handleEdit, canEdit } = props;
```

Comment:
1. Looks like we have unused imports
2. onClick is taking a return object "e" which is never used, may be we can remove it as well.

Reply:
@RohitKandimalla Thanks I removed those two.

Reply:
I am little confused on the implementation, Help me understand. Why does References are not editable ?
MeasureReference & MeasureDefinition both are sending a key prop to MeasureMetaDataRow but is never captured in child component. 

If we haven't worked on Editing references yet, then may be we can use "key" prop as intended or remove it

Reply:
@RohitKandimalla I just checked the code, looks like it's implemented but behind a flag, so i added the canEdit for references as well.

Reply:
Not the canEdit, The prop "key" 
MeasureMetaDataRowProps doesn't have a prop "key" but both the parent components are sending a prop "key" do we need it ?

Reply:
@RohitKandimalla I thought for a list we do need a key?

Reply:
we do but its never used

Reply:
@RohitKandimalla ok i removed it. thanks